src/plugins/data/common/field_formats/types.ts

```diff
@@ -17,19 +17,23 @@
  * under the License.
  */
 
+import { TEXT_CONTENT_TYPE, HTML_CONTENT_TYPE } from './content_types';
+
 /** @public **/
 export type ContentType = 'html' | 'text';
 
 /** @public **/
 export { IFieldFormat } from './field_format';
 
+interface HtmlConvertOptions {
+  field?: any;
+  hit?: Record<string, any>;
+  meta?: any;
```

Comment:
not sure that we need it, looks like meta === parsedURL

Reply:
Removed